test/scripts/status.test.js

```diff
@@ -0,0 +1,194 @@
+import init from "../../src/scripts/init.js";
+import addImplementation from "../../src/scripts/add-implementation.js";
+import sync from "../../src/scripts/sync.js";
+import newVersion from "../../src/scripts/new-version.js";
+import createProxy from "../../src/scripts/create-proxy.js";
+import status from "../../src/scripts/status.js";
+import setStdlib from "../../src/scripts/set-stdlib";
+import { FileSystem as fs } from "zos-lib";
+import { cleanup, cleanupfn } from "../helpers/cleanup.js";
+import TestLogger from '../helpers/logger.js';
+
+const should = require('chai')
+      .use(require('chai-as-promised'))
+      .should();
+
+contract('status command', function([_, owner]) {
+
+  const from = owner;
+  const txParams = { from };
+  const appName = "MyApp";
+  const contractName = "ImplV1";
+  const contractAlias = "Impl";
+  const anotherContractName = "AnotherImplV1";
+  const anotherContractAlias = "AnotherImpl";
+  const version = "0.1.0";
+  const network = "test";
+  const stdlibNameVersion = 'mock-stdlib@0.1.0';
+  const packageFileName = "test/tmp/package.zos.json";
+  const networkFileName = `test/tmp/package.zos.${network}.json`;
+  
+  const logger = new TestLogger();
```

Comment:
If we have the status checker we can return a result of the current status and check that result without having to test the logs... in that case, the script file could just output the result through the logs